deploy/README.md

```diff
@@ -1,27 +1,121 @@
+# Prerequisites
 
-Follow the [document](./deploy-infrastructure.md) to understand and execute the infrastructure deployment process.
+The deployment script uses following tools, please follow the links provided to install the suggested tools on your computer using which you would execute the script.
 
-## Executing the synapse pipeline
+- [git](https://github.com/git-guides/install-git) 
+- [az cli](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [jq](https://stedolan.github.io/jq/download/)
+- The scripts are executed on bash shell, so if using a computer with windows based operating system, install a [WSL](https://docs.microsoft.com/windows/wsl/about) environment to execute the script.
+- Install bicep on az cli using command `az bicep install`.
+- The bicep templates have been written to adhere to the syntax and rules for bicep version >= 0.8.9. Please check your bicep version using `az bicep version` if you run into bicep related errors.
+- The user performing the deployment of the bicep template and the associated scripts should have `Owner` role assigned at the subscription to which the resources are being deployed.
+- This solution assumes no interference from Policies deployed to your tenant preventing resources from being deployed. 
+- Get the repository to find the scripts. Clone the repository using following command.
 
-We have developed a sample Azure Synapse Pipeline demostrating the use of custom-vision ai-model to process a sample image.
+```bash
+git clone git@github.com:Azure/Azure-Orbital-Analytics-Samples.git
+```
 
-Follow the [document](./using-pipeline.md#executing-the-pipeline) to configure and execute the pipeline.
+Alternatively, one can use Azure Cloud Bash to deploy this sample solution in their Azure subscription.
+
+## Infrastructure Deployment
+
+Before executing the script one would need to login to azure using `az` cli and set the subscription in which they want to provision the resources.
+
+```bash
+az login
+az account set -s <subscription_id>
+```
+
+To install, configure and package the infrastructure for dependencies one can execute the script [setup.sh](./setup.sh)
+
+Script has been written to be executed with minimalistic input, it requires following input
+- `environmentCode` which serves as the prefix for infrastructure services names. Allows only alpha numeric(no special characters) and must be between 3 and 8 characters.
+- `location` which suggests which azure region infrastructure is deployed in.
+
+```bash
+./deploy/setup.sh <environmentCode> <location>
+```
+
+For eg.
+```bash
+./deploy/setup.sh aoi eastus
+```
+
+[setup.sh](./setup.sh) executes tasks in 3 steps
+- installs the infrastructure using [install.sh](./install.sh) script.
+- configures the infrastructure for setting up the dependecies using [configure.sh](./configure.sh) script.
+- packages the pipeline code to imported into synapse for standalone executions using [package.sh](./package.sh) script.
```

Comment:
I'd just say the package script will generate the custom vision model package and not have reference to importing into Synapse.